rai/node/rpc.cpp

```diff
@@ -1578,6 +1579,25 @@ void rai::rpc_handler::frontiers ()
 	response_errors ();
 }
 
+void rai::rpc_handler::is_confirmed ()
```

Comment:
I suggest something more consistent with other RPCs, such as
"block_confirmed"

Reply:
Should I also rename the rai::node and rai::ledger methods?